app/controllers/users_controller.rb

```diff
@@ -24,17 +30,15 @@ def index
 
   # GET /users/1
   # GET /users/1.json
-  def show
-  end
+  def show; end
 
```

Comment:
It should show the list of events that the user created. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/associations/user_show_requirement.png) from Odin)

It should list events previoulsy attendented and upcoming events  **using event model scope** ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/associations/user_events_requirement.png) from [Odin](https://www.theodinproject.com/courses/ruby-on-rails/lessons/associations))